core/rawdb/chain_freezer.go

```diff
@@ -282,6 +276,18 @@ func (f *chainFreezer) freeze(db ethdb.KeyValueStore) {
 				last = first + freezerBatchLimit
 			}
 		}
+
+		// check env first before chain freeze, it must wait when the env is necessary
+		if err := f.checkFreezerEnv(); err != nil {
+			f.waitEnvTimes++
+			if f.waitEnvTimes >= maxWaitFreezerEnvTimes {
+				log.Warn("Wait freezer env too many times, skip the chain freezing.")
```

Comment:
update the log message to avoid mis-understanding, like:
"Wait freezer env too many times, skip the chain freezing, not an issue if not import blocks(db compact, offline prune...)"